acceptance/bundle/validate/include_locations/output.dev.json

```diff
@@ -0,0 +1,100 @@
+{
+  "bundle": [
+    {
+      "column": 3,
+      "file": "databricks.yml",
+      "line": 2
+    }
```

Comment:
Not sure who is the consumer of this?

Did you consider a more compact format?

`"bundle": "databricks.yml:2:3" ` or `"bundle": ["databricks.yml:2:3"] `

I don't see any cases with multiple locations, is that possible?

Reply:
Re: multiple locations. This is possible in the case of overrides. I explicitly exclude those here but allow for multiple locations in the schema if we decide we need them later. Consumers can use the first element if they don't care.

Re: compact format. A downside of doing this is that consumers will reimplement logic to extract the same information that would be encoded here. Keeping it verbose cuts out the middle man.